lib/bot.js

```diff
@@ -390,6 +440,20 @@ class Bot {
       return match;
     });
 
+    // Webhooks first
+    const webhook = this.findWebhook(channel);
+    if (webhook) {
+      logger.debug('Sending message to Discord via webhook', withMentions, channel, '->', `#${discordChannel.name}`);
+      webhook.client.sendMessage(
```

Comment:
formatting nitpick:
```js
webhook.client.sendMessage(withMentions, {
  text,
  username: author,
  avatarUrl: this.getDiscordAvatar(author, channel)
})
```

(could even do `const avatarURL = this.getDiscordAvatar(author, channel);` first - always optimize for readability)